Cargo.toml

```diff
@@ -21,13 +20,27 @@ num-traits = "0.2"
 num-derive = "0.4"
 noop_proc_macro = "0.3.0"
 serde = { version = "1.0", features = ["derive"], optional = true }
-wasm-bindgen = { version = "0.2.63", optional = true }
 profiling = { version = "1" }
 tracing = { version = "0.1.40", optional = true }
 aligned-vec = "0.5.0"
 
-[dev-dependencies]
-criterion = { version = "0.5", features = ["html_reports"] }
+# Note: We need to differentiate between wasm32-unknown-unknown and
+# wasm32-wasi. cfg(target_arch = "wasm32") will match both.
+
+# We need to disable the default "rayon" feature on wasm32.
+[target.'cfg(not(target_arch = "wasm32"))'.dev-dependencies]
+criterion = "0.5"
+
+[target.'cfg(target_arch = "wasm32")'.dev-dependencies]
+criterion = { version = "0.5", default-features = false }
+
+# wasm-bindgen is needed for wasm32-unknown-unknown, but not
+# for wasm32-wasi.
+[target.wasm32-unknown-unknown.dependencies]
```

Comment:
We could, but I think that this is more explicit because we only want to enable it for this one specific target.

I would've used `target = "wasm32-unknown-unknown"` in the tests, but that is sadly not possible.

Reply:
I see, no problem then